drivers/bus/softqspi.c

```diff
@@ -168,9 +168,10 @@ STATIC void mp_soft_qspi_write_cmd_data(void *self_in, uint8_t cmd, size_t len,
 
 STATIC void mp_soft_qspi_write_cmd_addr_data(void *self_in, uint8_t cmd, uint32_t addr, size_t len, const uint8_t *src) {
     mp_soft_qspi_obj_t *self = (mp_soft_qspi_obj_t*)self_in;
-    uint8_t cmd_buf[4] = {cmd, addr >> 16, addr >> 8, addr};
+    uint8_t cmd_buf[4] = {cmd};
```

Comment:
This needs to be `cmd_buf[5]` in the case of 32-bit addressing (gcc 9 caught this as a compile error).